.github/workflows/build-deploy-docs.yaml

```diff
@@ -1,70 +1,55 @@
-name: Build and deploy the docs
+name: Build the docs
```

Comment:
File name is out of date (build-deploy-docs.yaml)

Reply:
I renamed it to `build-docs.yaml` but then reverted it because i dont know if there's something else i should do if i renamed this, but will rename it again.